lang/vi/events/github/meta.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+return [
+    'delete' => [
+        'title' => '🗑 <b>Webhook Deleted</b>',
+        'organization' => '🏢 Organization: <b>:organization</b>',
+        'full_name' => '📦 Repository: 🦑<b>:full_name</b>',
+        'sender' => '👤 Sender (triggered the event): <b>:sender</b>',
```

Comment:
I want to update lang for VI.

```suggestion
        'title' => '🗑 <b>Webhook đã được xóa</b>',
        'organization' => '🏢 Tổ chức: <b>:organization</b>',
        'full_name' => '📦 Kho lưu trữ: 🦑<b>:full_name</b>',
        'sender' => '👤 Người thực hiện sự kiện: <b>:sender</b>',
```